src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -317,6 +362,38 @@ export default function TransactionFeedV2() {
     [data?.transactions]
   )
 
+  useEffect(
+    function vibrateForNewlyCompletedTransactions() {
+      const isFirstPage = originalArgs?.endCursor === FIRST_PAGE_TIMESTAMP
+      if (isFirstPage && newlyCompletedTransactions) {
+        vibrateSuccess()
+      }
+    },
+    [newlyCompletedTransactions, originalArgs]
+  )
+
+  useEffect(
+    function vibrateForUnknownCompletedTransactions() {
+      const unknownCompletedTransactions = (data?.transactions || [])
+        .filter((tx) => tx.status === TransactionStatus.Complete)
+        .filter((tx) => !knownCompletedTransactionsHashes.includes(tx.transactionHash))
+
+      if (unknownCompletedTransactions.length) {
+        vibrateSuccess()
+      }
+    },
+    [data?.transactions, knownCompletedTransactionsHashes]
+  )
+
+  useEffect(
+    function updateKnownCompletedTransactions() {
+      if (data?.transactions.length) {
+        dispatch(updateKnownCompletedTransactionsHashes(data.transactions))
+      }
+    },
+    [data?.transactions]
+  )
```

Comment:
It seems we could skip this effect by reacting to the underlying RTK-Query API fetched action, within the transactions reducer, right?

Reply:
@jeanregisser Will investigate and get back to you on this!